omnibus/config/software/chefdk.rb

```diff
@@ -39,13 +39,9 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
+dependency "libffi" if debian?
+dependency "ruby"
+dependency "ruby-windows-devkit" if windows?
```

Comment:
No need to ask for devkit here.  If we stop shipping dev-kit, chefdk (the software definition) shouldn't care.  The project definition will add it as needed.